Documentation/README.md

```diff
@@ -1,16 +1,26 @@
 # Documentation  
-Within this folder, you will find detailed instructions on the various functions the Hypixel Public API allows for.
+Within this folder, you will find detailed instructions on the various functions the Hypixel Public API allows.
 
 ## Date and Time  
-Generally dates are stored as a Unix Epoch times in milliseconds.
+Generally, date-times are stored as Unix epoch times in milliseconds.

```

Comment:
Should I add a Unix Epoch time Converter Link for people? (https://www.epochconverter.com/)

Reply:
I don't think that is necessary. They should be able to find one on their own if they need it.

Reply:
alright